projects/ngneat/transloco-locale/README.md

```diff
@@ -0,0 +1,185 @@
+# Transloco Locale L10N
+
+This plugin provides localization(l10n) support for Transloco.
+
+> Localization refers to the adaptation of a product, application or document content to meet the language, cultural and other requirements of a specific target market (a locale).
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Setup](#setup)
+- [Configuration](#configuration-options)
+- [Setting Locale](#setting-locale)
+  - [Locale Translation Files](#translation-files-name)
+  - [Language Locale Mapping](#language-locale-mapping)
+  - [Manually Setting Locale](#manually-setting-locale)
+- [Service API](#service-api)
+- [Localization Pipes](#localization-pipes)
+  - [Date](#date-pipe)
+  - [Currency](#currency-pipe)
+  - [Decimal](#decimal-pipe)
+  - [Percent](#percent-pipe)
+  - [Browser Support](#browser-support)
+
+## Installation
+
+```
+npm i @ngneat/transloco-locale
+```
+
+### Setup
+
+Inject `TranslocoLocaleModule` along with `TranslocoModule` into `AppModule`:
+
+```typescript
+import { TranslocoLocaleModule } from '@ngneat/transloco-locale';
+
+@NgModule({
+  imports: [TranslocoModule, TranslocoPersistLangModule.init()],
```

Comment:
TranslocoPersistLangModule => TranslocoPersistLangModule

Reply:
TranslocoPersistLangModule => TranslocoLocaleModule, no?

Reply:
Yes, that's what I meant, thanks.